SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -79,15 +78,13 @@ public IActionResult NewOrReturningUser_Post(NewOrReturningUserViewModel viewMod
 
                 var userDataModel = userDataStore.LoadUserData(viewModel.Reference);

```

Comment:
Not your code, but this line looks a bit pointless and causes a database request.